frontend/src/pages/ActivityReport/Pages/topicsResources.js

```diff
@@ -54,7 +54,7 @@ const TopicsResources = ({
             <br />
             Enter the URL for OHS resource(s) used.
             {' '}
-            <a href="https://eclkc.ohs.acf.hhs.gov/">https://eclkc.ohs.acf.hhs.gov/</a>
+            <a target="_blank" rel="noreferrer" href="https://eclkc.ohs.acf.hhs.gov/">https://eclkc.ohs.acf.hhs.gov/</a>
```

Comment:
no adding `rel="noreferrer"` adds a security warning.

Reply:
Can we use `rel="noopener"` instead for the same security benefit? Especially since this is going to a sibling project in OHS we want our referral traffic to be captured properly.

https://web.dev/external-anchors-use-rel-noopener/

BTW - thank you for this context comment. It made the review super easy and I learned something new.